src/main/java/readAndWrite/MenuSingleRow.java

```diff
@@ -0,0 +1,38 @@
+package readAndWrite;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class MenuSingleRow {
+
+    private int menuRowNumber;
+    private String menuRowDescription;
+    private Map<String, Integer> availableOptionToAnswerOrType;
+
+    public MenuSingleRow(int menuRowNumber, String menuRowDescription, Map<String, Integer> availableOptionToAnswerOrType) {
+        this.menuRowNumber = menuRowNumber;
+        this.menuRowDescription = menuRowDescription;
+        if(availableOptionToAnswerOrType != null) {
+            this.availableOptionToAnswerOrType = new LinkedHashMap<>(availableOptionToAnswerOrType);
```

Comment:
Is there any important reason that you want the linkedHashMap so much ? If not this if statement is useless and decreases your performance, because linked hash map is way slower that the HashMap.